internal/cluster/boot.go

```diff
@@ -64,6 +64,11 @@ func TearDown() error {
 	return nil
 }
 
+// Update pulls down the most recent versions of the Docker images
+func Update() error {
+	return dockerComposePull()
```

Comment:
Could you wrap the error here? "updating docker images failed" or sth similar (see `TearDown`).

Reply:
It seems like duplication but can do.

Reply:
not duplication, but you're encapsulating errors and forbidding leaking internals up.